src/ol_openedx_rapid_response/README.rst

```diff
@@ -0,0 +1,67 @@
+ol_openedx_canvas_integration
+=============================
+
+Overview
+--------
+
+A django app plugin for edx-platform to enable ``Rapid Response`` reports download functionality in ``Instructor`` tab.
+
+
+**NOTE:**
+
+Since the edx-platform’s ``Instructor Dashboard`` does not support the plugin based tab extensions so we had to make some changes in the edx-platform itself to add ``Rapid Responses`` tab through this plugin.
+So, the ``edx-platform`` branch/tag you're using must include below commit for ``ol-openedx-rapid-response`` plugin to work properly:
+
+- https://github.com/mitodl/edx-platform/commit/f6c7cc6ea528819e8779e3137c806911a81a3198
+
+Installation
+------------
+
+You can install it into any Open edX instance by using any of the following methods:
+
+- To get the latest stable release from PyPI
+
+
+.. code-block::
+
+    pip install ol-openedx-rapid-response
+
+
+- To create a build locally and install it
+
+.. code-block::
+
+    # Open Terminal
+
+    1. Navigate to "open-edx-plugins" directory
+    2. Run ./pants build  # Do this if you haven’t run it already
+    3. Run ./pants package ::  # This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for all the "ol_openedx_*" plugins in "open-edx-plugins/src")
+    4. Install the "ol-openedx-rapid-response" using any of ".whl" or ".tar.gz" files generated in the above step
```

Comment:
Following the above comment

```suggestion
**Option 2: Build the package locally and install it

Follow these steps in a terminal on your machine:

1. Navigate to ``open-edx-plugins`` directory
2. If you haven't done so already, run ``./pants build``
3. Run ``./pants package ::``. This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for all the "ol_openedx_*" plugins in "open-edx-plugins/src")
4. Move/copy any of the ".whl" or ".tar.gz" files for this plugin that were generated in the above step to the machine/container running Open edX (NOTE: If running devstack via Docker, you can use ``docker cp`` to copy these files into your LMS container)
5. Run a shell in the machine/container running Open edX, and install this plugin using pip 
```